hoot-core/src/main/cpp/hoot/core/ops/NamedOp.cpp

```diff
@@ -80,21 +79,10 @@ void NamedOp::apply(OsmMapPtr& map)
     {
       boost::shared_ptr<OsmMapOperation> t(
         Factory::getInstance().constructObject<OsmMapOperation>(s));
-
-      QString initMessage =
-        QString("Applying operation %1 / %2")
-        .arg(QString::number(opCount))
-        .arg(QString::number(_namedOps.size()));
       boost::shared_ptr<OperationStatusInfo> statusInfo =
         boost::dynamic_pointer_cast<OperationStatusInfo>(t);
-      if (statusInfo.get() && !statusInfo->getInitStatusMessage().trimmed().isEmpty())
-      {
-        initMessage += ": " + statusInfo->getInitStatusMessage();
-      }
-      else
-      {
-        initMessage += ": " + s + " ...";
-      }
+
+      QString initMessage = _getInitMessage(s, opCount, statusInfo);
```

Comment:
The code removed was put into the `_getInitMessage` function to deduplicate below.